modules/test/playwright/tests/cookies-banner-web/cookiesBannerScript.spec.ts

```diff
@@ -118,7 +119,10 @@ test('@LPD-25701 Cookie Banner Script', async ({
 			await updateButton.click();
 		}
 
-		await waitForAlert(page);
+		await waitForAlert(
+			page,
+			`Success:Your request completed successfully.`
+		);
```

Comment:
I don't think this change is needed. `waitForAlert` [defaults to this text](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/utils/waitForAlert.ts#L22).

Reply:
I thought this too but was failing locally without it until I changed it. I'll revert it and see if I get any further errors.